VideoLocker/res/values/attrs.xml

```diff
@@ -6,4 +6,610 @@
         <attr name="itemVerticalPadding" format="dimension" />
         <attr name="iconDefaultSize" format="dimension" />
     </declare-styleable>
+
+    <declare-styleable name="IconView">
+        <attr name="iconColor" format="color" />
+        <attr name="iconName" format="enum">
+            <!-- We can't using IconValue indices as the enum values,
+                 as that would necessitate either an append-only
+                 policy to make them constant, or a recalculation of
+                 all greater indices upon the addition of a new
+                 IconValue in the middle of the list. Instead,
+                 we use the hash code of the name as an established
+                 and non-arbitrary mapping. -->
+            <enum name="fa_adjust" value="-1136693357" />
```

Comment:
@1zaman if the value is the hash code of the name,    1)did you check the hash collision?   2) should we use enum.name().hashcode()  instead of hardcoding it?